core/src/marker/markerManager.cpp

```diff
@@ -63,6 +64,19 @@ bool MarkerManager::setStyling(MarkerID markerID, const char* styling) {
     return true;
 }
 
+bool MarkerManager::setBitmap(MarkerID markerID, int width, int height, const unsigned int* bitmapData) {
```

Comment:
Hmm... https://github.com/tangrams/tangram-es/blob/marker-images/core/src/gl/texture.cpp#L28 **might** already do this for us!